src/runtime-rs/crates/hypervisor/src/qemu/cmdline_generator.rs

```diff
@@ -894,6 +1042,14 @@ impl<'a> QemuCmdLine<'a> {
         Ok(())
     }
 
+    pub fn add_ccw(&mut self, device_id: &str, path: &str) -> Result<()> {
```

Comment:
The name is wrong. This is adding a block device, not a bus. Please rename it to `add_block_device`.

Reply:
yep! I will rename it. Thanks!